src/components/ImageDetails/ImageDetails.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Input, Box, CloseButton } from '@chakra-ui/react';
+
+function ImageDetails({
+  index,
+  name,
+  preview,
+  // register,
+  defaultDescription,
+  removeImage,
+  removeDescription,
+  updateDescription,
+}) {
+  // let currentDescription;
+  // const changeDescription = e => {
+  //   updateDescription(index, e.target.value);
+  //   // updateDescription(index, e.target.value);
+  // };
+  const [currentDescription, setCurrentDescription] = useState();
```

Comment:
You can instead set the default value of `currrentDescription` to `defaultDescription` so you don't need a `defaultValue` in the description `Input` component.